types/tengitsui/index.d.ts

```diff
@@ -0,0 +1,332 @@
+// Type definitions for tengitsui 4.4
+// Project: https://github.com/1026385513/tengitsui
+// Definitions by: stackman <1026385513@qq.com>
```

Comment:
This should be a link to your github account, instead of an email, like so:
```ts
// Definitions by: stackman <https://github.com/1026385513>
```
If you fix this, then you can remove the `"dt-header": false` line in the tslint config file below.